docs/core/about.md

```diff
@@ -25,7 +25,7 @@ C#, Visual Basic, and F# languages can be used to write applications and librari
 
 .NET Core exposes APIs for many scenarios, a few of which follow:
 
-- Primitive types, such as [string][string] and [int][int].
+- Primitive types, such as [bool][bool] and [int][int].
```

Comment:
For the link to be resolved, line 36 has to be modified as well. It should be:
[bool]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/bool